External/FEXCore/Source/Interface/Core/CPUBackend.cpp

```diff
@@ -83,5 +83,16 @@ bool CPUBackend::IsAddressInCodeBuffer(uintptr_t Address) const {
   return false;
 }
 
+std::vector<Context::Context::JITRegionPairs> CPUBackend::GetJITRegions() const {
+  std::vector<Context::Context::JITRegionPairs> RegionPairs;
```

Comment:
Perhaps add:

```cpp
RegionPairs.reserve(CodeBuffers.size());
```

before the loop, so everything is allocated up front

Reply:
Resolved.